src/typescript-service.ts

```diff
@@ -270,8 +284,6 @@ export default class TypeScriptService {
                 const configuration = this.projectManager.getConfiguration(fileName);
                 configuration.prepare(this.connection).then(() => {
                     try {
-                        this.projectManager.syncConfiguration(configuration, this.connection); // resync after getting new config
```

Comment:
we won't touch placeholders because before parsing files we'll ensure they are fetched from VFS and will operate with the real data